scripts/tests/lnv2-module-test.sh

```diff
@@ -8,4 +7,4 @@ source scripts/_common.sh
 build_workspace
 add_target_dir_to_path
 
-tests all
```

Comment:
Did the `all` ever have any meaning?

Reply:
yes. we used to be able to run parts of the test suite seperately but that was removed.